### PR TITLE
chore: updated latest AFG and cleaned up go.mod

### PR DESCRIPTION
**Pull request by Baha-sk**
Signed-off-by: Baha Shaaban <baha.shaaban@securekey.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,18 +7,11 @@ module github.com/trustbloc/edge-core
 go 1.13
```
Comment (troyronda) [inline go.mod:7]: update to 1.15

**Review by fqutishat — APPROVED** [review]

**Review by troyronda — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Baha-sk) [reply go.mod]: done

File: go.mod [hunk 2]
```diff
@@ -4,21 +4,14 @@
...
 require (
-	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/go-kivik/couchdb/v3 v3.2.5
-	github.com/go-kivik/kivik/v3 v3.2.3
-	github.com/go-sql-driver/mysql v1.5.0
 	github.com/google/uuid v1.1.2
 	github.com/hashicorp/vault v1.2.1-0.20200911125421-dba37adcb55a
-	github.com/hyperledger/aries-framework-go v0.1.6-0.20210304143139-eb97711ddc73
+	github.com/hyperledger/aries-framework-go v0.1.6-0.20210306141947-8094fee88506
 	github.com/igor-pavlenko/httpsignatures-go v0.0.21
```
Comment (troyronda) [inline go.mod:13]: fyi: this has a newer version @llorllale

**Review by rolsonquadras — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -4,21 +4,14 @@
...
 
-go 1.13
+go 1.15
 
 require (
-	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/go-kivik/couchdb/v3 v3.2.5
-	github.com/go-kivik/kivik/v3 v3.2.3
-	github.com/go-sql-driver/mysql v1.5.0
 	github.com/google/uuid v1.1.2
```
Comment (troyronda) [inline go.mod:10]: fyi: this has a newer version: v1.2.0

File: go.mod [hunk 4]
```diff
@@ -4,21 +4,14 @@
...
 	github.com/hashicorp/vault v1.2.1-0.20200911125421-dba37adcb55a
```
Comment (troyronda) [inline go.mod:11]: fyi: this has a newer version: v1.6.3

File: go.mod [hunk 5]
```diff
@@ -4,21 +4,14 @@
...
-	github.com/hyperledger/aries-framework-go v0.1.6-0.20210304143139-eb97711ddc73
+	github.com/hyperledger/aries-framework-go v0.1.6-0.20210306141947-8094fee88506
 	github.com/igor-pavlenko/httpsignatures-go v0.0.21
-	github.com/piprate/json-gold v0.3.1-0.20201222165305-f4ce31c02ca3
+	github.com/piprate/json-gold v0.4.0
 	github.com/spf13/cobra v0.0.6
```
Comment (troyronda) [inline go.mod:15]: fyi: this has a newer version: v1.1.3

**Pull request merged**